### ISSUE TITLE
.gemrelease doesn't work with key and host

### ISSUE THREAD

**Issue by eprothro** [svenfuchs/gem-release#59]
running `gem build` followed by `gem push --key foo --host http://foo.host/private` works.

However with a .gemrelease
```yml
release:
  key: foo
  host: http://foo.host/private
```

Running `gem release` fails, with a 401 from my gem server:

```
...
Pushing gem to http://foo.host/private...
Authorization key is invalid
...
Tasks: TOP => release => release:rubygem_push
```

Are `host` and `key` not designed to work in this manner? Documentation leads me to believe they are.

**Comment by svenfuchs** [comment]
Thanks for the report, @eprothro, and sorry for not following up on this earlier.

I believe your issue should be addressed in `2.0.0.pre.1`, see https://github.com/svenfuchs/gem-release/pull/64

Specifically, there's now more flexibility in configuring gem-release. And things are tested better. See https://github.com/svenfuchs/gem-release/tree/sf-v2#configuration for configuration options.

I believe in your case it should be sufficient to use:

```
# top level keys, apply to all commands
key: foo
host: http://foo.host/private
```

or

```
export GEM_RELEASE_KEY=foo
export GEM_RELEASE_HOST=http://foo.host/private
```

I am going to close this here. Please let me know if things do not work as expected.

❤️

**Comment by eprothro** [comment]
Thanks @svenfuchs! I'll check it out and get back to you here if there's an issue.
